module3/ex11_tee/ex11_tee.py

```diff
@@ -2,13 +2,33 @@
 
 
 class Tee(object):
-    pass
+    def __init__(self, *fds):
+        self.fds = fds
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        for one_file in self.fds:
+            one_file.flush()
+            one_file.close()
+
+    def write(self, text):
+        for one_file in self.fds:
+            one_file.write(text)
+
+    def writelines(self, list_text):
+        for one_file in self.fds:
+            new_list = [line+"\n" for line in list_text[:-1]] + [list_text[-1]]
```

Comment:
This line should be moved outside of the for loop, there's no point to call it multiple times.